src/main/java/org/broadinstitute/hellbender/tools/copynumber/gcnv/CopyNumberPosteriorRecord.java

```diff
@@ -0,0 +1,42 @@
+package org.broadinstitute.hellbender.tools.copynumber.gcnv;
+
+import org.apache.commons.math3.util.FastMath;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.MathUtils;
+import org.broadinstitute.hellbender.utils.Utils;
+
+import java.util.Map;
+
+/**
+ * A record containing a copy number posterior distribution for a single interval
+ */
+public class CopyNumberPosteriorRecord {
+
+    /**
+     * Posterior distribution stores in log scale
+     */
+    private final Map<IntegerCopyNumberState, Double> copyNumberPosteriorDistribution;
+
+    /**
+     * Tolerated error for checking equality of doubles
+     */
+    private final static double TOLERATED_ERROR = 1e-4;
```

Comment:
`private static final`.  Perhaps `ERROR_TOLERANCE` is a little more conventional, as well.

Reply:
Done